examples/cross_entropy_benchmarking_example.py

```diff
@@ -0,0 +1,38 @@
+# coverage: ignore
```

Comment:
Add more documentation like our other examples. A reference to the paper where this comes from e.t.c. https://github.com/quantumlib/Cirq/blob/master/examples/bcs_mean_field.py is a good example.